draft-ietf-httpbis-digest-headers.md

```diff
@@ -1473,17 +1479,33 @@ using SHA algorithms for a range of encodings. Note that these are formatted as
 base64. This function could be adapted to other algorithms and should take into
 account their specific formatting rules.
 
+While the JSON objects in the examples are consistently formatted
+using the compact notation (no indentation) for single-key entries
+and 2-space indentation for multi-key entries,
+`Digest` is media-type agnostic and does not provide canonicalization
+algorithms for specific formats.
+
```

Comment:
Since the appendix will get removed, I think it would help to put text to this effect in the opening paragraph of Section 10 and 11. 

I suggest a slight rewording of the text for extra clarity: 

"Some examples include JSON objects in the payload data. For presentation purposes,  objects that fit completely within the line-length limits are presented on a single line using compact notation with no leading whitespace. Objects that would exceed line-length limits are presented across multiple lines (one line per key-value pair) with 2 spaced of leading indentation. `Digest` is media-type agnostic and does not provide canonicalization algorithms for specific formats. Examples of Digest are calculated inclusive of any whitespace."

Reply:
Reasonable, I'll do it;  but I'd avoid repeating things. Should we move §10 and §11 under a common "Example" section?

Reply:
That's what we had before and it makes the table of contents too hard to read.

Reply:
Ok, @LPardue if you like the current setup approve/merge